### ISSUE TITLE
Missing Vocabulary - HGNC (Human Gene Nomenclature (European Bioinformatics Institute)

### ISSUE THREAD

**Issue by KJDonovan-RS21** [OHDSI/Vocabulary-v5.0#677]
The vocabulary HGNC (Human Gene Nomenclature (European Bioinformatics Institute) is listed on the Athena vocabulary site https://athena.ohdsi.org/vocabulary/list as an optional download, but when selected no vocabulary or concepts are downloaded.  Is this vocabulary available from Athena or should I download it elsewhere and stand up as a custom vocabulary?

**Comment by KJDonovan-RS21** [comment]
@mik-ohdsi 

I think the process that incorporated the HGNC vocabulary in the OMOP Genomic vocabulary may have skipped or overlooked many HGNC symbols.  Within the OMOP Genomic vocabulary (downloaded yesterday), there are presently 19,338 distinct HGNC symbols represented - See query below.

SELECT COUNT( DISTINCT SUBSTR(CONCEPT_NAME,1,INSTR(CONCEPT_NAME,' ')-1)) AS SYMBOL_COUNT
FROM PDS.CONCEPT WHERE VOCABULARY_ID='OMOP Genomic'; Returns-> 19338 Records

But when I download the complete HGNC vocabulary from https://www.genenames.org/download/custom/ I see 43,193 approved HGNC symbols.

If I am missing something here let me know, but there appears to be a significant gap.

Thanks, Kevin.

**Comment by mik-ohdsi** [comment]
I haven't looked into it in detail, but one fact that might contribute to this is that the current representation of genomic information is predominantly cancer centered and as such, a pre-selection of genes and variants for all participating terminologies, not only HGNC, might apply.

**Comment by cgreich** [comment]
@KJDonovan-RS21:

You are right, we only have a subset of what the HGNC puts out. We made the initial assumptions that we are mostly interested in the genes with a protein product. Most of them (about half) are. The rest are other sequences with or without a distinct function (locus type in HGNC lingo):

Locus type | Count
-- | --
gene with protein product | 19,252
pseudogene | 13,811
RNA, long non-coding | 5,602
RNA, micro | 1,912
RNA, transfer | 591
RNA, small nucleolar | 568
immunoglobulin gene | 230
T cell receptor gene | 206
immunoglobulin pseudogene | 203
readthrough | 140
RNA, cluster | 119
fragile site | 116
endogenous retrovirus | 109
unknown | 80
RNA, ribosomal | 60
RNA, small nuclear | 50
protocadherin | 39
region | 38
T cell receptor pseudogene | 37
RNA, misc | 30
complex locus constituent | 29
virus integration site | 8
RNA, vault | 4
RNA, Y | 4

Any good idea what we would need from a clinical research perspective? Remember, we are not building a sequence analysis pipeline. Our use case is understanding patient disease.
